packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -58,20 +58,20 @@ export function isHostClientMobile(): boolean {
 /**
  * @hidden
  * Helper function which indicates if current API is supported on mobile or not.
- * @returns SdkError if host client is not android/ios or if the requiredVersion is not
+ * @throws SdkError if host client is not android/ios or if the requiredVersion is not
  *          supported by platform or not. Null is returned in case of success.
  *
  * @internal
  */
-export function isApiSupportedOnMobile(requiredVersion: string = defaultSDKVersionForCompatCheck): SdkError {
+export function throwExceptionIfApiIsNotSupported(requiredVersion: string = defaultSDKVersionForCompatCheck): void {
```

Comment:
I realize you've changed the name of this function a lot, and I'm sorry that we've been going back and forth on it a lot. I think this name is much closer to correct, but what if we did "throwExceptionIfMobileApiIsNotSupported". The function seems intended to verify both that you are on a mobile device AND that the SDK version matches what you say is required.